sinon/lib/call.py

```diff
@@ -0,0 +1,23 @@
+'''
+Copyright (c) 2017, Kir Chou
```

Comment:
You can write your name here! :)

Reply:
It's actually better for your project if the whole thing's copyright is under one person's name. In the future, it will be easier for you if you want to re-release the software under a different license, create a closed-source fork or whatever. Since Sinon.PY is being released under an open source license, I'm fine with assigning you the copyright.

I will add myself as an author though. Thanks!